File: Pset-1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(){
    int cents;
    do{
        float cash = get_float("Change owed: ");
        cents = round(cash * 100);
    }
    while (cents < 0);
    
    int quarters = floor(cents / 25);
    int dimes = floor((cents % 25) / 10);
    int nickels = floor(((cents % 25) % 10) / 5);
    int pennies = floor(((cents % 25) % 10) % 5);
    
    int sum = quarters + dimes + nickels + pennies;
    printf("%d\n", sum);
}

File: Pset-6/cash.py
from math import floor


#get the number of cash from the user
while True:
    cash = float(input("Change owed: "))
    if cash >= 0:
        break 

#convert to cents
cents = round(cash * 100)

quarter = floor(cents / 25)
dimes = floor((cents % 25) / 10)
nickels = floor(((cents % 25) % 10) / 5)
pennies = floor(((cents % 25) % 10) % 5)

print(quarter + dimes + nickels + pennies)
